app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -416,18 +426,27 @@ public Player(@NonNull final MainPlayer service) {
         windowManager = ContextCompat.getSystemService(context, WindowManager.class);
     }
 
+    @NonNull
     private VideoPlaybackResolver.QualityResolver getQualityResolver() {
         return new VideoPlaybackResolver.QualityResolver() {
             @Override
-            public int getDefaultResolutionIndex(final List<VideoStream> sortedVideos) {
+            public int getDefaultResolutionIndex(final StreamInfo streamInfo,
+                                                 final List<VideoStream> sortedVideos) {
+                if (StreamTypeUtil.isLiveStream(streamInfo.getStreamType())) {
+                    return -1;
+                }
                 return videoPlayerSelected()
                         ? ListHelper.getDefaultResolutionIndex(context, sortedVideos)
                         : ListHelper.getPopupDefaultResolutionIndex(context, sortedVideos);
             }
 
             @Override
-            public int getOverrideResolutionIndex(final List<VideoStream> sortedVideos,
+            public int getOverrideResolutionIndex(final StreamInfo streamInfo,
+                                                  final List<VideoStream> sortedVideos,
                                                   final String playbackQuality) {
+                if (StreamTypeUtil.isLiveStream(streamInfo.getStreamType())) {
+                    return -1;
+                }
```

Comment:
... and here in my comment in the other review

Reply:
Outdated